tests/lint/multiqc_config.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+
+import yaml
+
+import nf_core.lint
+
+
+def test_multiqc_config_exists_ignore(self):
+    """Test that linting fails if the multiqc_config.yml file is missing"""
```

Comment:
should fail or ignore the test?

Reply:
In the code it is set to "ignored", no idea why, so I just copied that.

Reply:
It's not mandatory to use multiqc, so it makes sense, even though we could skip the test with the `.nf-core.yml` 🤔